src/main/scala/expressd/ExpressEnv.scala

```diff
@@ -46,7 +46,7 @@ object ExpressEnv {
 
   val BIAS_WINDOW = System.getProperty("express.biasWindow", "8").toInt
   val BIAS_MATRIX_SIZE = BIAS_WINDOW * 2 + 1
-  val MAX_FRAG_LENGTH = System.getProperty("express.maxFragLength", "800").toInt
+  val MAX_FRAG_LENGTH = System.getProperty("express.maxFragLength", "10000").toInt
```

Comment:
Note that I had to add a larger value here because default invocation of bowtie was allowing inserts of more than 1MB.

If we really want to avoid these impossibly large fragments, perhaps they should be filtered out with info message in ExpressD.scala.